src/content/docs/en/developers/guides/running-a-scroll-node.mdx

```diff
@@ -0,0 +1,405 @@
+        "storage": {
+            "0x0000000000000000000000000000000000000000000000000000000000000000": "0x18960EEc21b1878C581937a14c5c3C43008F6b6B",
+            "0x0000000000000000000000000000000000000000000000000000000000000001": "0x8ac7230489e80000",
+            "0x0000000000000000000000000000000000000000000000000000000000000002": "0xBa50f5340FB9F3Bd074bD638c9BE13eCB36E603d",
+            "0x0000000000000000000000000000000000000000000000000000000000000003": "0x2351C7aD0c8cFEB25c81301EAC922ab1f1980bbe"
+        }
+        }
+    },
+    "number": "0x0",
+    "gasUsed": "0x0",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "baseFeePerGas": null
+    }
+  ```
+</ToggleElement>
+
+---
+## Troubleshooting
```

Comment:
I think the first two could be removed, they seem to be irrelevant now

Reply:
I'm just going to remove the entire Troubleshooting section then